ArmPkg/ArmPkg.dec

```diff
@@ -110,6 +110,12 @@
   # Include/Guid/ArmMpCoreInfo.h
   gArmMpCoreInfoGuid = { 0xa4ee0728, 0xe5d7, 0x4ac5,  {0xb2, 0x1e, 0x65, 0x8e, 0xd8, 0x57, 0xe8, 0x34} }
 
+  # MU_CHANGE [BEGIN]
+  ## SMMU config data
+  # Include/Guid/SmmuConfig.h
+  gSmmuConfigGuid = { 0xcd56ec8f, 0x75f1, 0x440a, { 0xaa, 0x48, 0x09, 0x58, 0xb1, 0x1c, 0x9a, 0xa7 } }

```

Comment:
Not necessary by any means, but if this is only going to be used as a HOB GUID, ending with `HobGuid` is common to clarify usage. So, `gSmmuConfigHobGuid`. But, that's just used in some cases for clarity and it's fine to keep it as-is.

Reply:
done